Sources/PrimerSDK/Classes/Data Models/PrimerSettings.swift

```diff
@@ -12,7 +12,8 @@ public class PrimerSettings: PrimerSettingsProtocol, Codable {
 
     static var current: PrimerSettings {
         let settings: PrimerSettingsProtocol = DependencyContainer.resolve()
-        return settings as! PrimerSettings
+        guard let primerSettings = settings as? PrimerSettings else { fatalError() }
```

Comment:
Out of curiosity, did Swiftlint suggest this change?

Reply:
If we want to refactor this further, we just need to eliminate all fatalError() from our code. The point of the ticket is to fix the code writing format with linter, that is why I did it like this.

Reply:
If we want to refactor this further, we just need to eliminate all fatalError() from our code. The point of the ticket is to fix the code writing format with linter, that is why I did it like this.

Reply:
I think it's fine to get us past this stage, but we should revisit this very soon.

We need to think critically about why we're aligning to the swiftlint rules. 

If we're changing from 'bad code' to 'bad code' to make swiftlint happy, when it's shouting at us "this code is bad!", then we're not really making changes in the spirit of its purpose.

Reply:
One of the purposes is to silence potential swiftlint warnings when merchant includes our SDK in their project, that is the primary goal, right? Everything else should be seen as a bonus.

I agree we should make this code better, let us revisit this soon.